### PR TITLE
Some fixes while integrating into lotus

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: actors/builtin/codes.go [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
 func init() {
 	builder := cid.V1Builder{Codec: cid.Raw, MhType: mh.IDENTITY}
-	makeBuiltin := func(s string) abi.ActorCodeID {
+	makeBuiltin := func(s string) cid.Cid {
 		c, err := builder.Sum([]byte(s))
 		if err != nil {
 			panic(err)
 		}
-		id := abi.ActorCodeID(c)
+		id := cid.Cid(c)
```
Comment (anorth) [inline actors/builtin/codes.go]: unnecessary conversion?

File: actors/builtin/singletons.go [hunk 2]
```diff
@@ -8,12 +8,12 @@ import (
...
-	RewardActorAddr        = mustMakeAddress(2)
-	CronActorAddr          = mustMakeAddress(3)
-	StoragePowerActorAddr  = mustMakeAddress(4)
-	StorageMarketActorAddr = mustMakeAddress(5)
+	InitActorAddr          = mustMakeAddress(0)
+	RewardActorAddr        = mustMakeAddress(1)
+	StoragePowerActorAddr  = mustMakeAddress(2)
+	StorageMarketActorAddr = mustMakeAddress(3)
+	CronActorAddr          = mustMakeAddress(4)
+	SystemActorAddr        = mustMakeAddress(5)
```
Comment (anorth) [inline actors/builtin/singletons.go]: This seems like an unnecessary change to me. The proposed ordering is somewhat arbitrary history-dependent order, the original somewhat builds up from foundations to application-level.

File: actors/builtin/multisig/multisig_actor.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ func (a *MultiSigActor) State(rt vmr.Runtime) (vmr.ActorStateHandle, MultiSigAct
 }
 
 type ConstructorParams struct {
-	Signers     []addr.Address
```
Comment (frrist) [inline actors/builtin/multisig/multisig_actor.go]: Thanks for reminding me that CI needs a gofmt check. https://github.com/filecoin-project/specs-actors/issues/42

File: actors/builtin/singletons.go (hunk 2 above)
Comment (whyrusleeping) [reply actors/builtin/singletons.go]: I worry that users in the community already have things set up to interact with system actors. Forcing that change will be disruptive, but I won't push the point any further

File: actors/builtin/codes.go (hunk 1 above)
Comment (whyrusleeping) [reply actors/builtin/codes.go]: weird that gofmt -s didnt catch that...
